### ISSUE TITLE
Episode images missing?

### ISSUE THREAD

**Issue by bitcrumb** [krestaino/podcast-xml-parser#7]
There seems to be no way of fetching episode images?

**Comment by krestaino** [comment]
Added support in `3.0.0-beta.2`. Give it a go: `npm install podcast-xml-parser@3.0.0-beta.2`. Please note, in version `3.x.x` there is no default export, so you need to update your code:

### Version `2.x.x`
`import podcastXmlParser from "podcast-xml-parser";`

### Version `3.x.x`
`import { podcastXmlParser } from "podcast-xml-parser";`

**Comment by bitcrumb** [comment]
@krestaino I tried it out but doesn't seem to be working?

Could it be that `channel` should be `item` in this line of code?

```
itunesImage: getAttribute(channel, "itunes:image.@_href"),
```

Looking at the rest of the parsing code I think this should be:

```
itunesImage: getAttribute(item, "itunes:image.@_href"),
```

**Comment by krestaino** [comment]
Whoops! Yeah, you're right, I should be using `item` not `channel`. Fixed in `3.0.0-beta.3`.

**Comment by bitcrumb** [comment]
Thanks for your quick response! 🥳
